pyomo/contrib/gdpopt/config_options.py

```diff
@@ -9,60 +9,23 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-from pyomo.common.config import (ConfigValue, NonNegativeInt, 
-                                 In, PositiveInt, NonNegativeFloat,
-                                 ConfigBlock, ConfigList)
+from pyomo.common.config import (ConfigBlock, ConfigList, ConfigValue,
+                                 In, NonNegativeFloat, NonNegativeInt,
+                                 PositiveInt)
 from pyomo.contrib.gdpopt.master_initialize import valid_init_strategies
-from pyomo.contrib.gdpopt.util import _DoNothing, a_logger
-
-
-def _get_GDPopt_config():
-    _supported_strategies = {
-        'LOA',  # Logic-based outer approximation
-        'GLOA',  # Global logic-based outer approximation
-        'LBB',  # Logic-based branch-and-bound
-        'RIC',  # Relaxation with Integer Cuts
-    }
-    CONFIG = ConfigBlock("GDPopt")
-    CONFIG.declare("iterlim", ConfigValue(
-        default=100, domain=NonNegativeInt,
-        description="Iteration limit."
-    ))
-    CONFIG.declare("time_limit", ConfigValue(
-        default=600,
-        domain=PositiveInt,
-        description="Time limit (seconds, default=600)",
-        doc="Seconds allowed until terminated. Note that the time limit can "
-            "currently only be enforced between subsolver invocations. You may "
-            "need to set subsolver time limits as well."
-    ))
-    CONFIG.declare("strategy", ConfigValue(
-        default=None, domain=In(_supported_strategies),
-        description="Decomposition strategy to use."
-    ))
-    CONFIG.declare("tee", ConfigValue(
-        default=False,
-        description="Stream output to terminal.",
-        domain=bool
-    ))
-    CONFIG.declare("logger", ConfigValue(
-        default='pyomo.contrib.gdpopt',
-        description="The logger object or name to use for reporting.",
-        domain=a_logger
-    ))
-    _add_OA_configs(CONFIG)
-    _add_BB_configs(CONFIG)
-    _add_subsolver_configs(CONFIG)
-    _add_tolerance_configs(CONFIG)
-    return CONFIG
-
+from pyomo.contrib.gdpopt.nlp_initialization import (
+    restore_vars_to_original_values)
+from pyomo.contrib.gdpopt.util import _DoNothing
+from pyomo.opt import SolverFactory, UnknownSolver
+from pyomo.solvers.plugins.solvers.persistent_solver import PersistentSolver
 
 def _add_OA_configs(CONFIG):
     CONFIG.declare("init_strategy", ConfigValue(
         default="set_covering", domain=In(valid_init_strategies.keys()),
```

Comment:
Since the `strategies` were rebranded as `algorithms`, should the initialization ones be renamed as well?

Reply:
Yes, I think this makes sense.